source/administration/config-settings.rst

```diff
@@ -1608,6 +1608,9 @@ Enable cross-origin requests from
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Enable HTTP cross-origin requests from specific domains separated by spaces. Type ``*`` to allow CORS from any domain or leave it blank to disable it.
 
+.. note::
+ Please make sure you have entered your SiteURL before enabling this setting to prevent losing access to the System Console after saving.  If you do experience lost access to the System Console after changing this setting, you can set your `SiteURL <https://docs.mattermost.com/administration/config-settings.html#site-url>`_ through the config.json file.
```

Comment:
- `SiteURL` to `Site URL`
- `If you do experience..` to `If you experience`
- in-line code `config.json` with double backticks